dynawo/sources/Modeler/DataInterface/DYNGeneratorInterfaceIIDM.h

```diff
@@ -129,8 +129,25 @@ class GeneratorInterfaceIIDM : public GeneratorInterface, public InjectorInterfa
    */
   int getComponentVarIndex(const std::string& varName) const;
 
+  /**
+   * @brief Getter for the bus' country
+   * @return the bus country
+   */
+  inline const std::string& getCountry() const {
+    return country_;
+  }
+
+  /**
+   * @brief Setter for the bus' country
+   * @param country bus country
+   */
+  inline void setCountry(const std::string& country) {
+    country_ = country;
+  }
+
```

Comment:
Adapt comment. bus' country -> load' country

Reply:
copy paste is bad

Reply:
Done